contributors/design-proposals/cluster-lifecycle/draft-20171020-bootstrap-checkpointing.md

```diff
@@ -0,0 +1,108 @@
+# Kubernetes Bootstrap Checkpointing Proposal
+
+## Metadata
+
+```yaml
```

Comment:
This is neat. I assume it was auto-generated somehow.

Reply:
Nope, it's a modified kep from @calebamiles .  The original kep 's vernacular was a little weird imo and I may PR against the template.